index.html

```diff
@@ -0,0 +1,163 @@
+                <li><a href="$"><img class="facebook" src="./cap-detials/facebook.png" alt="facebook"></a></li>
+                <li><a href="$"><img class="twitter" src="./cap-detials/twitter.png" alt="twitter"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+            </ul>
+        </nav>
+
+        <nav class="desktop-nav-2 flex">
+            <a href="./index.html"><img src="./cap-detials/fogo.png" alt="Logo"></a>
+            <ul class="flex">
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#program">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#nominees">Nominees</a></li>
+                <li><a href="#partners">Partners</a></li>
+                <li><button type="button">Vote Now</button></li>
+            </ul>
+            
+        </nav>
+    </header>
+
+    <main>
+        <section class="home-intro flex">
+            <div class="intro-heading">
+                <h2>"Hello there! Footballl Lovers"</h2>
+                <h1>Ziggies<br> Award Ceremony <br> 2022</h1>
+            </div>
+            <p class="mobile-paragraph"> Ziggies Award ceremony is an award ceremony to crown the best player for the 2022 season which takes place in France.
+               our Award Ceremony brings together top coaches,Players & journalists from all over the world for a unique and joyful experience.</p>
+            <h2 class="mobile-date">2022.02.20(Fri)</h2>
+            <p class="mobile-location">@ La Fonte Event Center, Paris,France</p>
+        </section>
+
+        <section class="mainprogram flex" id="program">
+            <div class="header flex">
+                <h3>Main Program</h3>
+                <hr>
+            </div>
+
+            <div class="activities flex">
+                <div class="activity flex">
+                    <div class="flex imgiconwrapper">
+                        <img src="./cap-detials/program_icon_01.png" alt="programicon">
+                        <h3>Arrival</h3>
+                    </div>
+                    <p>the special guest and staffs arrive for the ceremony.</p>
+                </div>
+
+                <div class="activity flex">
+                    <div class="flex imgiconwrapper">
+                        <img src="./cap-detials/program_icon_02.png" alt="programicon">
+                        <h3>Media</h3>
+                    </div>
+                    <p>special guests attend to journalists and the press,take pictures etc.</p>
+                </div>
+
+                <div class="activity flex">
+                    <div class="flex imgiconwrapper">
+                        <img src="./cap-detials/program_icon_03.png" alt="programicon">
+                        <h3>Opening ceremony</h3>
+                    </div>
+                    <p>Everyone will get attend the Opening ceremony in the main hall.</p>
+                </div>
```

Comment:
- [ ]  The HTML uses semantic tags, which means,: :innocent:  sections use `section`, articles use `article`..., Kindly for each activity place them in article tags and not divs. `<article class="program-list"></article>` 👍🏻 It could be something like this :point_down: guess that make sense 😅 

```
        <article class="activity flex">
             <div class="flex imgiconwrapper">
                 <img src="./cap-detials/program_icon_03.png" alt="programicon">
                 <h3>Opening ceremony</h3>
             </div>
             <p>Everyone will get attend the Opening ceremony in the main hall.</p>
       </article>
```